docs/contributing/how-to-open-a-pull-request.md

```diff
@@ -103,7 +101,7 @@ For the Gatsby blog, it's necessary to run your content idea by the Gatsby team
 
 ## Follow up with reviews and suggestions
 
-After a PR is sent to the Gatsby GitHub repo, the Gatsby core team and the community may suggest modifications to the changes that your PR introduces.
```

Comment:
I get that for some parts of managing a PR only the core team can do, but technically anyone can review a PR so this sounds like the only opinions that matter are those of Gatsby Inc.

Reply:
👍 I like that nuance. We definitely welcome and encourage the community to weigh in, so I'll leave this in since the _management_ of PRs is covered elsewhere.

Thanks for the catch!